reconcile/utils/terrascript_client.py

```diff
@@ -3493,6 +3493,10 @@ def populate_tf_resource_alb(self, resource, namespace_info,
             'tags': common_values['tags'],
             'depends_on': self.get_dependencies([sg_tf_resource]),
         }
+
+        if 'idle_timeout' in resource:
+            values['idle_timeout'] = resource['idle_timeout']
```

Comment:
If this comes from a graphql query this path will always exist, see https://issues.redhat.com/browse/APPSRE-3868 . In that case, this is equivalent:
```python
        values['idle_timeout'] = resource['idle_timeout']
```

and, if we fix the task above, then 
```suggestion
        values['idle_timeout'] = resource.get('idle_timeout')
```

Reply:
Ah yes I forgot about that graphql behavior 

I adjusted the condition in 0d91787